### PR TITLE
extract audio code in audio utils. Update README

### PR DESCRIPTION
**Pull request by wirthual**
Follow up changes for the CLAP functionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,6 +230,7 @@ async def embed(engine: AsyncEmbeddingEngine):
 
 asyncio.run(embed(array["laion/clap-htsat-unfused"]))
 ```
+*Note:* Audio Files need to have a sample rate of 48kHz.
```
Comment (michaelfeil) [inline README.md:233]: Already fixed in https://github.com/michaelfeil/infinity/pull/374 @wirthual #375 

The sampling rate needs to be the same as the model, not generally 48k

File: libs/infinity_emb/infinity_emb/transformer/audio/utils.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import requests
+
+from infinity_emb._optional_imports import CHECK_REQUESTS, CHECK_SOUNDFILE
+from infinity_emb.primitives import (
+    AudioCorruption,
+    AudioSingle,
+)
+
+if CHECK_REQUESTS.is_available:
+    import requests  # type: ignore
```
Comment (michaelfeil) [inline libs/infinity_emb/infinity_emb/transformer/audio/utils.py:13]: Sorry, I made a concurrent PR!

**Comment by michaelfeil** [comment]
@wirthual Thanks a lot for your contributions! Seems like I ran in some concurrency issues working on the same issues as you were. I just refactored the requests section into async requests.

Happy to use a similar, if its based on the latest main

**Comment by wirthual** [comment]
Your as fast as your server ;) Let me open a new PR

**Pull request closed without merging**